docs/dev/development.md

```diff
@@ -29,37 +29,42 @@ If you want to contribute changes to Cirq, you will instead want to fork the rep
 1. Fork the Cirq repo (Fork button in upper right corner of
 [repo page](https://github.com/quantumlib/Cirq)).
 Forking creates a new github repo at the location
-```https://github.com/USERNAME/cirq``` where ```USERNAME``` is
+https://github.com/USERNAME/cirq where `USERNAME` is
 your github id.
 1. Clone the fork you created to your local machine at the directory
 where you would like to store your local copy of the code, and `cd` into the newly created directory.
-    ```bash
+    
+   ```bash
     git clone git@github.com:USERNAME/cirq.git
     cd Cirq
-    ```
-    (Alternatively, you can clone the repository using the URL provided
-    on your repo page under the green "Clone or Download" button)
+   ```
+   (Alternatively, you can clone the repository using the URL provided on your repo page under the green "Clone or Download" button)
 1. Add a remote called ```upstream``` to git.
```

Comment:
Make this single backtick?

Similar for every other triple backtick where its not needed.